data_transformation.py

```diff
@@ -73,53 +72,67 @@ def count_emoji(text):
 
 
 def get_name_from_flag(data):
-    return pycountry.countries.get(flag=data).name if pycountry.countries.get(flag=data) else data
+    return pycountry.countries.get(flag=data).alpha_2 if pycountry.countries.get(flag=data) else data
 
 
 dict_countries = {
-    'usa': 'us',
-    'united states': 'us',
-    'us': 'us',
-    'u.s.': 'us',
-    'united states of america': 'us',
-    'estados unidos': 'us',
-    'estados unidos de américa': 'us',
-    'united kingdom': 'gb',
-    'gb': 'gb',
-    'great britain': 'gb',
-    'britain': 'gb',
-    'british isles': 'gb',
-    'reino unido': 'gb',
-    'gran bretaña': 'gb',
-    'england' : 'gb'
+    'USA': 'US',
+    'UNITED STATES': 'US',
+    'US': 'US',
+    'U.S.': 'US',
+    'UNITED STATES OF AMERICA': 'US',
+    'ESTADOS UNIDOS': 'US',
+    'ESTADOS UNIDOS DE AMÉRICA': 'US',
+    'UNITED KINGDOM': 'GB',
+    'UK': 'GB',
+    'GREAT BRITAIN': 'GB',
+    'BRITAIN': 'GB',
+    'BRITISH ISLES': 'GB',
+    'REINO UNIDO': 'GB',
+    'GRAN BRETAÑA': 'GB',
+    'ENGLAND' : 'GB',
+    'LONDON' : 'GB',
 } #TODO complete with the most frequent countries names
 
 
 def unify_countries_name(country_name):
-    country_name = country_name.lower()
+    country_name = country_name.upper()
     if country_name in dict_countries:
         return dict_countries[country_name]
     else:
         return country_name
 
 
-list_countries = ['USA', 'UK', 'india'] #TODO complete with the most frequent countries
+list_countries = ['USA', 'UK', 'INDIA', 'LONDON', 'ENGLAND', 'FRANCE', 'CANADA'] #TODO complete with the most frequent countries
 
 
 def extract_country(text):
     for country in list_countries:
         pattern = r'\b{}\b'.format(country)
         search_pattern = re.search(pattern, text, flags=re.IGNORECASE)
-        return search_pattern.group() if search_pattern else text
+        if search_pattern:
+            return search_pattern.group()
+    return text
+
+
+def get_iso_2(text):
```

Comment:
I'd change function name for something more descriptive, such as `get_iso_2_for_country`